style.css

```diff
@@ -0,0 +1,264 @@
+body {
+  padding: 0%;
+  margin: 0%;
+}
+
+/* minimum mobile size 375px */
+
+@media screen and (min-width: 375px) {
+  /* header style */
+
+  #header {
+    height: 7vh;
+    position: top;
+    display: flex;
+    padding: 3px;
+    align-items: center;
+  }
+
+  #navigation {
+    position: relative;
+    width: 100%;
+    padding: 0%;
+  }
+
+  #navigation ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    text-decoration: none;
+    align-items: center;
+    list-style: none;
+    text-align: center;
+  }
+
+  #logo {
+    width: 30%;
+    height: 20px;
+    text-decoration: none;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    text-align: center;
+    color: #6070ff;
+    margin-left: -18px;
+  }
+
+  #menu-icon {
+    width: 30%;
+    height: 24px;
+  }
+
+  .menu-link {
+    margin-right: 10px;
+    width: 30px;
+    height: 30px;
+  }
+
+  /* Main section Style */
+
+  .presentation {
+    width: 100%;
+    height: 92vh;
+    background-image: url("./images/headerBackground.svg");
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-position: top;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .greeting {
+    width: 70%;
+    height: 150px;
+    margin: 130px 24px 24px 0;
+    margin-left: 24px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 23px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    position: relative;
+    flex-grow: 0;
```

Comment:
- [Optional] Great job for  using the exact font as given in Figma design, :+1: however your font is not applying because you didn't import it, so please import it from [google font](https://fonts.google.com/) either in your HTML or CSS file to be applied, check [this](https://www.freecodecamp.org/news/how-to-use-google-fonts-in-your-next-web-design-project-e1ad48f1adfa/) for more info :+1: